docs/markdown/Shipping-prebuilt-binaries-as-wraps.md

```diff
@@ -42,6 +42,33 @@ compiler flags) might not be compatible. If you do this, then you are
 responsible for verifying that your libraries are compatible, Meson
 will not check things for you.
 
+## Using a wrap file
+
+To make this all work automatically, a project will need a
+[wrap file](Wrap-dependency-system-manual#wrap-format), as well as the
```

Comment:
The link destination needs to be the .md file (this will be rewritten to the same file extension as the output format, i.e. html)